### ISSUE TITLE
Updating PoB deleted my old builds 

### ISSUE THREAD

**Issue by liniim** [PathOfBuildingCommunity/PathOfBuilding#7939]
### Check version

- [X] I'm running the latest version of Path of Building and I've verified this by checking the [changelog](https://github.com/PathOfBuildingCommunity/PathOfBuilding/blob/master/CHANGELOG.md)

### Check for duplicates

- [X] I've checked for duplicate open **and closed** issues by using the search function of the [issue tracker](https://github.com/PathOfBuildingCommunity/PathOfBuilding/issues?q=is%3Aissue)

### What platform are you running Path of Building on?

Windows

### How is Path of Building expected to behave?

Updating PoB usually does not affect the build files.

### How does Path of Building behave?

I updated PoB after not doing so for over a year (since ~2.25 or since POE 3.21 Crucible League). After updating all my builds in \Builds were gone. I had a nested folder, \Builds\Imported Builds, that was not affected and the builds inside were still there.

### Screenshots

![image](https://github.com/user-attachments/assets/6a2f6985-df7a-4218-b6b5-e7bfa2689d84)

![image](https://github.com/user-attachments/assets/845c918f-9fa1-4733-a375-d36bdd0ae3cf)

**Comment by Nightblade** [comment]
I think old versions of pob stored builds in a different directory than newer versions?

Is is possible your old builds are in a different directory?

**Comment by liniim** [comment]
I don't think so because my custom folder and the files in that are still there.

**Comment by LocalIdentity** [comment]
@liniim Maybe try to use a tool like “everywhere” to make sure the folder or builds are not located somewhere else

**Comment by liniim** [comment]
Have tried it, no  dice. 

![image](https://github.com/user-attachments/assets/2e0f066c-c9f2-44d6-9409-fcbf672fb4f2)

**Comment by LocalIdentity** [comment]
@liniim try and search for the name of a build as the folder is not always named "Builds"

**Comment by Nightblade** [comment]
A long shot I know, but have you checked the Recycle Bin?

**Comment by Nightblade** [comment]
Closing as stale / no feedback.
